java/Oleksii-Riabov/src/com/zoolatech/lecture3/tasks/_1/Instrument.java

```diff
@@ -0,0 +1,19 @@
+package com.zoolatech.lecture3.tasks._1;
+
+public class Instrument {
```

Comment:
```
Design classes that represent ... musical instruments (at least 3 different one)
```

note, that `name` and `sound` shouldn't been defined as properties of these classes; imagine, it's a simplified behaviour of different instruments

Reply:
Solve this by defining Instrument as interface.